### PR TITLE
Split Object Browser and Buckets Admin

### PR DESCRIPTION
**Pull request by dvaldivia**
Split the Object Browser and the Buckets Admin page

Screenshot 2022-12-12 at 12 12 37 PM

Screenshot 2022-12-12 at 12 12 39 PM

Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

**Comment by bexsoft** [comment]
https://github.com/dvaldivia/console/pull/5

File: .github/workflows/jobs.yaml [hunk 1]
```diff
@@ -16,207 +16,6 @@ concurrency:
...
-          echo "start ---> make test-operator-integration";
-          make test-operator-integration;
-
-      - uses: actions/cache@v3
-        id: coverage-cache-operator
-        name: Coverage Cache Operator
-        with:
-          path: |
-            ./operator-integration/coverage/
-          key: ${{ runner.os }}-coverage-2-operator-${{ github.run_id }}
```
Comment (reivaj05) [inline .github/workflows/jobs.yaml:219]: Are these tests no longer needed?

Comment (reivaj05) [reply .github/workflows/jobs.yaml:219]: Nvm, they went to the bottom 🤦

**Review by reivaj05 — APPROVED** [review]

File: .github/workflows/jobs.yaml (hunk 1 above)
Comment (dvaldivia) [reply .github/workflows/jobs.yaml:219]: hahaha yeah, I re-arranged the file

**Comment by cniackz** [comment]
Tested locally:

image

**Review by bexsoft — APPROVED** [review]

**Comment by cniackz** [comment]
All is looking good, my only concern is the failure at test:

Screenshot 2022-12-14 at 6 17 55 PM

But maybe this isn't related as we are just changing the UI, right?.

In any case, the log is:

```
    user_api_bucket_test.go:2825: 
        	Error Trace:	/home/runner/work/console/console/integration/user_api_bucket_test.go:2825
        	Error:      	Not equal: 
        	            	expected: 204
        	            	actual  : 500
        	Test:       	TestReplication
        	Messages:   	Http Response: {"code":500,"detailedMessage":"The remote target does not exist","message":"an error occurred, please try again"}
--- FAIL: TestReplication (0.51s)
```

and the code around is:

```go
	// 7. Delete remaining Bucket Replication Rule with generic end point:
	// /buckets/{bucket_name}/delete-all-replication-rules
	response, err = DeletesAllReplicationRulesOnABucket(
		originBucket,
	)
	assert.Nil(err)
	if err != nil {
		log.Println(err)
		return
	}
	finalResponse = inspectHTTPResponse(response)
	if response != nil {
		assert.Equal(204, response.StatusCode, finalResponse)
	}
```

We can open another PR to fix this test

**Review by cniackz — APPROVED** [review]
LGTM

**Pull request merged**